src/plugins/template/src/config.cpp

```diff
@@ -83,6 +83,50 @@ Configuration::Configuration(const ov::AnyMap& config, const Configuration& defa
             log_level = value.as<ov::log::Level>();
         } else if (ov::hint::model_priority == key) {
             model_priority = value.as<ov::hint::Priority>();
+        } else if (ov::optimal_batch_size == key) {
+            try {
+                int tmp_i = value.as<int>();
+                if (tmp_i >= 0)
+                    optimal_batch_size = tmp_i;
+                else
+                    OPENVINO_THROW("Incorrect value, it should be unsigned integer: ", key);
+            } catch (const std::exception&) {
+                OPENVINO_THROW("Wrong value ", value.as<std::string>(), "for property key ", key);
+            }
+        } else if (ov::max_batch_size == key) {
+            try {
+               int tmp_i = value.as<int>();
+                if (tmp_i >= 0)
+                    max_batch_size = tmp_i;
+                else
+                    OPENVINO_THROW("Incorrect value, it should be unsigned integer: ", key);
+            } catch (const std::exception&) {
+                OPENVINO_THROW("Wrong value ", value.as<std::string>(), "for property key ", key);
+            }
+        } else if (ov::loaded_from_cache == key) {
+            loaded_from_cache = value.as<bool>();
+        } else if (ov::enable_mmap == key) {
+            enable_mmap = value.as<bool>();
+        } else if (ov::compilation_num_threads == key) {
```

Comment:
The same comment as for optimal_batch_size

Reply:
done